### ISSUE TITLE
Chocolatey package internalizer fails for erlang package

### ISSUE THREAD

**Issue by lnonce** [chocolatey-beam/erlang-package#1]
When running the choco package internalizer `choco download --recompile erlang` it fails because it is not able to resolve the download urls.

```
Chocolatey v1.1.0 Business Downloading existing package(s) to 
C:\jenkins\workspace\Chocolatey\Update_choco-selfservice-test_from_Community_Repository\0809efed-3c8b-4411-915a-c052c04
58e2a\download 
erlang v26.0.2 (forced) [Approved] 
Found internalizable Chocolatey functions. Inspecting values for 
remote resources. Unable to find value for variable '$otp_version'. May not be able to download resource from url 
'https://github.com/erlang/otp/releases/download/OTP-$otp_version/otp_win32_$otp_version.exe'. Unable to find value for
 variable '$otp_version'. May not be able to download resource from url 
```

This is due to the way how the variables are declared in the chocolateyInstall.ps1. The choco download command does not recognize the `New-Variable -Name otp_version...` style declarations.

Would it be possible to change it to standard variable assignments as it used to be in older package versions (e.g. 25.0.4)?
```
$otp_version = '@@OTP_VERSION@@'
$erts_version = '@@ERTS_VERSION@@'
```

Or another option would be to use the placeholder in the url:
```
Url = "https://github.com/erlang/otp/releases/download/OTP-@@OTP_VERSION@@/otp_win32_@@OTP_VERSION@@.exe"
```

Also happy to submit a PR if desired.
Greetings

**Comment by lukebakken** [comment]
Please feel free to submit a PR, thanks.

**Comment by lukebakken** [comment]
Fixed by #2
